browser/static/js/main_track.js

```diff
@@ -788,7 +788,7 @@ function handle_mousemove(evt) {
 function handle_mouseup(evt) {
   mousedown = false;
   if (edit_mode) {
-    //send click&drag coordinates to caliban.py to update annotations
+    //send click&drag coordinates to reviews.py to update annotations
```

Comment:
This got changed back to `caliban.py` from `reviews.py`
```suggestion
    //send click&drag coordinates to caliban.py to update annotations
```